templates/superwerker.template.yaml

```diff
@@ -33,6 +37,8 @@ Metadata:
         default: Domain for automated DNS configuration
       Subdomain:
         default: Subdomain for automated DNS configuration
+      NotificationsMail:
+        default: Mail address used for notifications
```

Comment:
Maybe even `for important notification regarding your AWS account`
and
`leave empty for no notifications`